README.rst

```diff
@@ -415,12 +415,12 @@ following code to your app::
 
   from djangosaml2.signals import pre_user_save
 
-  def custom_update_user(sender=user, attributes=attributes, user_modified=user_modified)
+  def custom_update_user(sender=User, instance, attributes, user_modified, **kargs)
      ...
      return True  # I modified the user object
 
 
-Your handler will receive the user object, the list of SAML attributes
+Your handler will receive the user object instance, the list of SAML attributes
```

Comment:
I was trying to clarify that the signal handler will receive the User object in the variable named 'instance'